### PR TITLE
libraries/tlsh: updated for version 4.12.0

### PR DESCRIPTION
**Pull request by mattegger**

### CODE REVIEW

File: libraries/tlsh/tlsh.SlackBuild [hunk 1]
```diff
@@ -28,7 +28,7 @@
 cd $(dirname $0) ; CWD=$(pwd)
 
 PRGNAM=tlsh
-VERSION=${VERSION:-4.8.2}
+VERSION=${VERSION:-4.12.0}
 BUILD=${BUILD:-2}
```
Comment (willysr) [inline libraries/tlsh/tlsh.SlackBuild:33]: You forgot to reset the BUILD

**Review by willysr — CHANGES_REQUESTED** [review]
please check the BUILD

File: libraries/tlsh/tlsh.SlackBuild [hunk 2]
```diff
@@ -28,8 +29,8 @@
 cd $(dirname $0) ; CWD=$(pwd)
 
 PRGNAM=tlsh
-VERSION=${VERSION:-4.8.2}
-BUILD=${BUILD:-2}
+VERSION=${VERSION:-4.12.0}
+BUILD=${BUILD:-1}
```
Comment (mattegger) [inline libraries/tlsh/tlsh.SlackBuild:33]: corrected build number

**Comment by willysr** [comment]
merged

**Pull request closed without merging**